### PR TITLE
Current story

### PR DESCRIPTION
**Pull request by MHG16**
Please check out CurrentStory and give me a thumbs up if it looks ok.

### CODE REVIEW

File: public/scripts/components/pages/Details.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import React from 'react';
+import CurrentStory from '../sub-components/CurrentStory.js';
 
 export default React.createClass({
 	render: function() {
+		console.log(this.props.params);
```
Comment (NicerHugs) [inline public/scripts/components/pages/Details.js]: pls remove `console.log`

File: public/scripts/components/sub-components/CurrentStory.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Model from '../../models/StoryModel.js';
+
+export default React.createClass({
+	
+	getInitialState: function () {
+	  	console.log('getInitialState');
```
Comment (NicerHugs) [inline public/scripts/components/sub-components/CurrentStory.js]: pls remove `console.log`

File: public/scripts/components/sub-components/CurrentStory.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	  	
+		return {story: new Model({id: this.props.id})};
+	 },
+
+	componentDidMount: function () {
+		console.log(this.state);
```
Comment (NicerHugs) [inline public/scripts/components/sub-components/CurrentStory.js]: pls remove `console.log`

File: public/scripts/components/sub-components/CurrentStory.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+		this.state.story.fetch();
+		this.state.story.on('change', (updatedStory) => {
```
Comment (NicerHugs) [inline public/scripts/components/sub-components/CurrentStory.js]: pls add code to remove this event listener when the component unmounts.

**Comment by MHG16** [comment]
I removed all console.log()s and added code to remove the event listener when the component unmounts

File: public/scripts/components/sub-components/CurrentStory.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+			this.setState({story: updatedStory});
+
+		});	
+	},
+
+	componentWillUnmount: function () {
+		this.state.story.removeEventListener('change', (updatedStory));
```
Comment (NicerHugs) [inline public/scripts/components/sub-components/CurrentStory.js]: you want to use the backbone event (un)listener found here: http://backbonejs.org/#Events-off

**Comment by alarner** [comment]
👍

**Comment by NicerHugs** [comment]
fix the merge conflicts then 👍

**Comment by MHG16** [comment]
Please check and give me a thumbs up if ok.  I added code to remove the event listener when the component unmounts.

**Comment by ihatetoast** [comment]
👍

**Comment by alarner** [comment]
👍

**Comment by NicerHugs** [comment]
👍

**Pull request merged**